packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Stepper.doc.ts

```diff
@@ -14,6 +15,9 @@ const data: ReferenceEntityTemplateSchema = {
   ],
   category: 'Components',
   related: [],
+  defaultExample: {
+    codeblock: generateCodeBlock('Stepper', 'stepper', 'default.example'),
```

Comment:
Obviously theres no general rule but I started following JS's helper function pattern. What's your opinion? I know it may be unnecessary here since theres only one example but it might be good for consistency sake if the team uses the same approach

```
const generateCodeBlockForStepper = (title: string, fileName: string) =>
  generateCodeBlock(title, 'stepper', fileName);
```

Reply:
Yeah, I think as long as it's only being called once, it's unnecessary. Once it's called two or more times, it's nice.